ui/js/compareperf.js

```diff
@@ -201,9 +201,13 @@ perf.controller('CompareResultsCtrl', [
                     if (rsid == 'original') {
                         $scope.originalResultSetID = resultSet.id;
                         $scope.originalTimestamp = resultSet.push_timestamp;
+                        $scope.originalAuthor = resultSet.author;
+                        $scope.originalComments = resultSet.comments;
                     } else {
                         $scope.newResultSetID = resultSet.id;
                         $scope.newTimestamp = resultSet.push_timestamp;
+                        $scope.newAuthor = resultSet.author;
+                        $scope.newComments = resultSet.comments;
```

Comment:
If you wanted to, you could probably simplify this to:

$scope.newResultSet = resultSet.

And then replace all calls that reference newResultSetID, newTimeStamp, etc. with accesses to the object properties.

Same goes for originalResultSet<foo> above.

Up to you whether you want to do this.